src/textField/index.tsx

```diff
@@ -46,6 +46,8 @@ export interface TextFieldProps extends CommonFormFields {
   label?: string
   buttonTitle?: string
   focus?: boolean
+  inputMode?: string
```

Comment:
You should be a bit more restrictive on this field, as we do for types. 
https://developer.mozilla.org/fr/docs/Web/HTML/Attributs_universels/inputmode

Reply:
and export `inputMode` as static like @mcellier was saying for `type`
https://github.com/blablacar/ui-library/pull/276/files#r297204113